docs/user_guide/workloads/run_ray_cluster.md

```diff
@@ -0,0 +1,84 @@
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## Note
+This example demonstrates how to set up [KubeRay](https://docs.ray.io/en/master/cluster/kubernetes/getting-started.html) and run a [Ray Cluster](https://docs.ray.io/en/master/cluster/kubernetes/getting-started/raycluster-quick-start.html) with the YuniKorn scheduler. It relies on an admission controller to configure the default applicationId and queue name.
+[Yunikorn supported labels](https://yunikorn.apache.org/docs/user_guide/labels_and_annotations_in_yunikorn) and [Yunikorn queue setting](https://yunikorn.apache.org/docs/user_guide/queue_config)
+
+## Modify YuniKorn settings
+Follow [YuniKorn install guide](https://yunikorn.apache.org/docs/) and modify YuniKorn configmap "yunikorn-defaults" to allow ray operator based on k8s service account.
+```
+kubectl patch configmap yunikorn-defaults -n yunikorn --patch '{"data":{"admissionController.accessControl.systemUsers": "^system:serviceaccount:kube-system:,^system:serviceaccount:default:"}}' 
+```
+
+## Setup a KubeRay operator
+```
+helm repo add kuberay https://ray-project.github.io/kuberay-helm/
+helm repo update
+helm install kuberay-operator kuberay/kuberay-operator --version 1.1.1
+```
+- The result should be as shown below
+![ray_cluster_operator](../../assets/ray_cluster_operator.png)
+
+## Create Ray Cluster 
+```
+helm install raycluster kuberay/ray-cluster --version 1.1.1
+```
+- Ray Cluster result
+  ![ray_cluster_cluster](../../assets/ray_cluster_cluster.png)
+- YuniKorn UI
+  ![ray_cluster_on_ui](../../assets/ray_cluster_on_ui.png)
+### Configure your Ray Cluster(optional)
+If you disable admission controller, you need to add the schedulerName: yunikorn in [raycluster spec](https://github.com/ray-project/kuberay/blob/master/helm-chart/ray-cluster/templates/raycluster-cluster.yaml#L40).By applicationId label, pods with same id are marked under same application or all pods from raycluster CRD share a application.
+```
+#example
+metadata:
+  labels:
+    applicaionId: ray-cluster-0001
+    queue: root.ray.clusters
+spec:
+  schedulerName: yunikorn # k8s will inform yunikorn based on this
+```
+
+
+## Submit a RayJob to Ray Cluster
+```
+export HEAD_POD=$(kubectl get pods --selector=ray.io/node-type=head -o custom-columns=POD:metadata.name --no-headers)
+echo $HEAD_POD
+
+kubectl exec -it $HEAD_POD -- python -c "import ray; ray.init(); print(ray.cluster_resources())"
+```
+
+Services in Kubernetes aren't directly accessible by default. However, you can use port-forwarding to connect to them locally.
+```
+kubectl port-forward raycluster-kuberay-head-svc 8265:8265
+```
+After port-forward set up, you can access the Ray dashboard by going to http://localhost:8265/ in your web browser.
```

Comment:
Lint:
<img width="476" alt="image" src="https://github.com/apache/yunikorn-site/assets/26764036/4e785068-59e9-4a4d-baad-14d29b81bd6e">

Change to ->  "going to \`http://localhost:8265` in your web browser."

Result:
<img width="445" alt="image" src="https://github.com/apache/yunikorn-site/assets/26764036/156a85c7-9ff0-4bc1-8a82-0a3eb43d12cd">